cql/fhir-cql/src/main/java/com/ibm/fhir/cql/engine/searchparam/SearchParameterResolver.java

```diff
@@ -37,8 +43,8 @@ public SearchParameter getSearchParameterDefinition(String resourceType, String
             path = "";
         }
 
-        // XXX should this use the registry (all parameters) or the filtered set of search parameters for a given tenant?
-        Map<String, SearchParameter> params = SearchUtil.getSearchParameters(resourceType);
+        // should this use the registry (all parameters) or the filtered set of search parameters for a given tenant?
```

Comment:
Is this an open question, or is it documenting the behavior?

Reply:
i think its an open question, but unrelated to the PR.  i don't think its too important